[2023-11-08T08:01:50.158+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract.dbt_run scheduled__2023-11-07T00:00:00+00:00 [queued]>
[2023-11-08T08:01:50.171+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract.dbt_run scheduled__2023-11-07T00:00:00+00:00 [queued]>
[2023-11-08T08:01:50.171+0000] {taskinstance.py:1361} INFO - Starting attempt 4 of 4
[2023-11-08T08:01:50.186+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): dbt_run> on 2023-11-07 00:00:00+00:00
[2023-11-08T08:01:50.196+0000] {standard_task_runner.py:57} INFO - Started process 281 to run task
[2023-11-08T08:01:50.203+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract', 'dbt_run', 'scheduled__2023-11-07T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmpstexqt_b']
[2023-11-08T08:01:50.207+0000] {standard_task_runner.py:85} INFO - Job 21: Subtask dbt_run
[2023-11-08T08:01:50.296+0000] {task_command.py:416} INFO - Running <TaskInstance: extract.dbt_run scheduled__2023-11-07T00:00:00+00:00 [running]> on host e37a8383d3d6
[2023-11-08T08:01:50.422+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='extract' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2023-11-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-07T00:00:00+00:00'
[2023-11-08T08:01:50.427+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-08T08:01:50.428+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt run --project-dir /app/airflow/dbt_nba']
[2023-11-08T08:01:50.445+0000] {subprocess.py:86} INFO - Output:
[2023-11-08T08:01:56.011+0000] {subprocess.py:93} INFO - [0m08:01:56  Running with dbt=1.7.0
[2023-11-08T08:01:58.648+0000] {subprocess.py:93} INFO - [0m08:01:58  Registered adapter: bigquery=1.7.0
[2023-11-08T08:01:58.676+0000] {subprocess.py:93} INFO - [0m08:01:58  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2023-11-08T08:02:00.510+0000] {subprocess.py:93} INFO - [0m08:02:00  [[33mWARNING[0m]: Did not find matching node for patch with name 'my_first_dbt_model' in the 'models' section of file 'models/example/schema.yml'
[2023-11-08T08:02:00.675+0000] {subprocess.py:93} INFO - [0m08:02:00  Encountered an error:
[2023-11-08T08:02:00.675+0000] {subprocess.py:93} INFO - Compilation Error
[2023-11-08T08:02:00.676+0000] {subprocess.py:93} INFO -   Model 'model.nba_stats.my_second_dbt_model' (models/example/my_second_dbt_model.sql) depends on a node named 'my_first_dbt_model' which was not found
[2023-11-08T08:02:01.748+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-08T08:02:01.785+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/root/.cache/pypoetry/virtualenvs/02-dbt-in-airflow-BsnH11MF-py3.8/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-08T08:02:01.794+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=extract, task_id=dbt_run, execution_date=20231107T000000, start_date=20231108T080150, end_date=20231108T080201
[2023-11-08T08:02:01.826+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 21 for task dbt_run (Bash command failed. The command returned a non-zero exit code 2.; 281)
[2023-11-08T08:02:01.851+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-08T08:02:01.883+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
